FishyTransport/Sources/FishyActorsCore/Analysis.swift

```diff
@@ -15,20 +15,24 @@
 import SwiftSyntax
 import Foundation
 
-final class Analysis: SyntaxVisitor {
+// very naive pretty printing
+let Bold = "\u{001B}[0;1m"
+let Reset = "\u{001B}[0;0m"
+
+public final class Analysis: SyntaxVisitor {
 
   private let sourceDirectory: String
   private let verbose: Bool
 
-  var decls: [DistributedActorDecl] = []
+  public var decls: [DistributedActorDecl] = []
```

Comment:
please don't make vars public like that;

There should rather be a function that allows returns this -> the run() could return this or there could be another getAnalysedDecls or something

Reply:
I removed public here since it wasn't needed anymore. Since Analysis can't be accessed from other modules do you think it's still valuable to add a method to access it? I like the idea of making decls private and having run() return it, do you think that's a better approach?

Reply:
> I like the idea of making decls private and having run() return it, do you think that's a better approach?

Yeah sounds good :)